nixos/modules/services/web-apps/atlassian/jira.nix

```diff
@@ -170,7 +170,6 @@ in
       description = "Atlassian Jira";
 
       wantedBy = [ "multi-user.target" ];
-      requires = [ "postgresql.service" ];
```

Comment:
requires = mkIf services.postgresql.enable [ "postgresql.service" ];

Reply:
Instead of `cfg` this should be `config`.